src/mapblock_mesh.cpp

```diff
@@ -1104,7 +1107,7 @@ MapBlockMesh::MapBlockMesh(MeshMakeData *data, v3s16 camera_offset):
 		Convert FastFaces to MeshCollector
 	*/
 
-	MeshCollector collector(m_use_tangent_vertices);
```

Comment:
Does this mean `m_use_tangent_vertices` did not have any noticeable effect when `enable_bumpmapping` and `enable_parallax_occlusion` were both used?